### PR TITLE
[DOCS] Fixes URLs on Secure communication with APM Agents page

### PR DESCRIPTION
**Pull request by szabosteve**
## Overview

This PR fixes a bad link that breaks the build.

### CODE REVIEW

**Comment by szabosteve** [comment]
@elasticmachine update branch

**Review by axw — APPROVED** [review]
LGTM. This means we'll lose the .NET Agent link until the docs are fixed, but better that than block all the builds.

**Comment by axw** [comment]
@bmorelli25 we'll need to replace the .NET Agent link later. Seems the 1.8.0 release broke the docs builds.

File: docs/secure-communication-agents.asciidoc [hunk 1]
```diff
@@ -481,8 +481,8 @@ Each Agent has a configuration for setting the value of the secret token:
 To enable secure communication in your Agents, you need to update the configured server URL to use `HTTPS` instead of `HTTP`.
 
 * *Go Agent*: {apm-go-ref}/configuration.html#config-server-url[`ELASTIC_APM_SERVER_URL`]
-* *Java Agent*: {apm-java-ref}/config-reporter.html#config-server-urls[`server_urls`]
-* *.NET Agent*: {apm-dotnet-ref}/config-reporter.html#config-server-urls[`ServerUrls`]
+* *Java Agent*: {apm-java-ref}/config-reporter.html#config-server-url[`server_urls`]
```
Comment (szabosteve) [inline docs/secure-communication-agents.asciidoc:484]:
```suggestion
* *Java Agent*: {apm-java-ref}/config-reporter.html#config-server-urls[`server_urls`]
```

**Pull request merged**